docs/architecture/adr-044-lite-client-with-weak-subjectivity.md

```diff
@@ -129,13 +302,15 @@ Accepted.
 
 ### Positive
 
-* light client which is safe to use (it can go offline, but not for too long)
+* light client which is safe to use
 
 ### Negative
 
 * complexity of bisection
 
 ### Neutral
 
-* social consensus can be prone to errors (for cases where a new light client
-  joins a network or it has been offline for too long)
+* if the light client has been offline for `duration > TrustPeriod`, a user
+  will have to reinitialize it
+* evidence propagation delay prevents the light client from immediately
```

Comment:
Not sure we can reason that way. We can think about user defined policy on when to consider header as trusted (for example waiting some time, receiving header from multiple full nodes, etc), but in any case this does not guarantee that a user is still seeing forged header. Therefore, we always act after the fact and by having these mechanisms incentivise correct behaviour so that user can trust the verified header.